src/docs/app-monitoring/Sysdig-Monitoring-Guidline-for-shared-services-and-apps.md

```diff
@@ -0,0 +1,276 @@
+
+page_purpose: Documented monitoring metrics and monitoring tools that will describe the approach to monitoring and alerting that will be applied to all platform-shared services and apps.
+
+audience: developer, technical lead
+
+author: Billy Li
+
+content_owner: Billy Li
+
+sort_order: 2
+---
+
+## Intro
+
+SRE, which stands for Site Reliability Engineering, has become an important concept to ensure an application's reliability by quickly returning the system to a steady state. Software can be used as a tool to gain full visibility into the health of an application or system, allowing us to identify and resolve issues before they impact stakeholders.
+
+In this document, we will explore the basic concept of SRE and demonstrate how to set it up using the Registry application as an example.
+
+
+#### Registry
+The registry is an application where teams can submit requests for provisioning namespaces in OpenShift 4 (OCP4) clusters. The registry allows teams to:
+
+- Update project contact details and other metadata;
+- Request that their project namespace be created in additional clusters;
+- Request other resources such as KeyCloak realms or Artifactory pull-through repositories; and
+- Receive management from the platform services team.
+
+More details about the Registry app and its workings can be found [here](https://github.com/bcgov/platform-services-registry/blob/master/docs/Whole-project-workflow.md).
+
+### Setting up SRE
+The scope of SRE encompasses the deployment, configuration, and monitoring of the app. It also covers the availability, latency, change management, emergency response, and capacity management of services in production. To ensure optimal performance and reliability, we use several tools and methodologies that align with SRE principles.
+
+
+#### SLA
+The client should be at the center of every aspect of your customer agreement. An incident may need fixing ten different issues on the back end. But from the client's perspective, the only thing that matters is that the system works as it should, and facts should be reflected in your SLAs and SLOs. Be sure to limit your promises to high-level, user-facing functionality, and always use plain language in SLAs. Based on this, we can come up with some SLA for the Registry:
+
+* Normal users can successfully load the Registry dashboard within 5 seconds.
+* Admin user can successfully load Registry dashboard within 13 seconds.
+* An approved product request can be provisioned within an hour.
+* Application is up online 99% of the time.
+* Update product requests can be processed within an hour.
+
+
+#### SLO 
+The definition of s SLO is a service level objective: a target value or range of values for a service level that is measured by an SLI. Google has a really [good doc](https://sre.google/workbook/implementing-slos/#:~:text=For%20example%2C%20if%20you%20have,50%25%20of%20the%20error%20budget.) for how to gets the start.
```

Comment:
```suggestion
The definition of SLO is a service level objective: a target value or range of values for a service level that is measured by an SLI. Google has a really [good doc](https://sre.google/workbook/implementing-slos/#:~:text=For%20example%2C%20if%20you%20have,50%25%20of%20the%20error%20budget.) for how to gets the start.
```

Reply:
based on the google link, it's important to know how to calculate error budget if ppl were to use the error budget–based approach for reliability, can you provide more details on that end?